### PR TITLE
Adds examine tips to firedoor construction/deconstruction

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: You can examine a firedoor for hints on how to
construct/deconstruct it.
add: Add an additional hint during construction where you can add
plasteel to make the firedoor reinforced.
/:cl:

- Makes welding a firedoor take the same amount of time as welding an
airlock.
- Makes the firelock construction/destruction steps equivalent to each
other, specifically, changed a construction weld step to a crowbar step.

### CODE REVIEW

File: code/game/machinery/doors/firedoor.dm [hunk 1]
```diff
@@ -30,6 +30,17 @@
 	..()
 	CalculateAffectingAreas()
 
+/obj/machinery/door/firedoor/examine(mob/user)
+	..()
+	if(!density)
+		to_chat(user, "<span class='notice'>It is open, but could be <b>pried</b> closed.</span>")
+	else if(density && !welded)
```
Comment (ChangelingRain) [inline code/game/machinery/doors/firedoor.dm]: all of these check for density despite being under the `else` for `if(!density)`; they don't need to check for density.

Comment (coiax) [inline code/game/machinery/doors/firedoor.dm]: It's more readable that way.

Comment (ChangelingRain) [inline code/game/machinery/doors/firedoor.dm]: All of these check for density despite being under the `else` for `if(!density)`; make them not check for density, as it is already checked.

**Review by ChangelingRain — DISMISSED** [review]

File: code/game/machinery/doors/firedoor.dm (hunk 1 above)
Comment (coiax) [inline code/game/machinery/doors/firedoor.dm]: I'll modify the if's structure.

**Comment by coiax** [comment]
@ChangelingRain I did the changes, merge plox?

**Pull request merged**